common/src/main/java/com/wynntils/models/quests/QuestModel.java

```diff
@@ -144,107 +131,36 @@ public void openQuestOnWiki(QuestInfo questInfo) {
-    }
-
-    private void updateTrackedQuest(QuestInfo questInfo) {
-        trackedQuest = questInfo;
-        WynntilsMod.postEvent(new TrackedQuestUpdateEvent(trackedQuest));
-    }
-
-    private Optional<QuestInfo> getQuestInfoFromName(String name) {
+    public Optional<QuestInfo> getQuestInfoFromName(String name) {
         List<QuestInfo> questInfoList = name.startsWith(MINI_QUEST_PREFIX) ? miniQuests : quests;
 
         return questInfoList.stream()
                 .filter(quest -> quest.getName().equals(stripPrefix(name)))
                 .findFirst();
     }
 
-    private boolean updateAfterRescan(String name, StyledText nextTask) {
-        boolean isMiniQuest = name.startsWith(MINI_QUEST_PREFIX);
-        List<QuestInfo> questInfoList = isMiniQuest ? miniQuests : quests;
-
-        if (questInfoList.isEmpty()) {
-            afterRescanTask = nextTask;
-            afterRescanName = stripPrefix(name);
-            rescanQuestBook(!isMiniQuest, isMiniQuest);
-            return true;
-        }
-
-        return false;
-    }
-
     private String stripPrefix(String name) {
         return StringUtils.replaceOnce(name, MINI_QUEST_PREFIX, "");
     }
 
-    void updateQuestsFromQuery(List<QuestInfo> newQuests, QuestInfo trackedQuest) {
+    void updateQuestsFromQuery(List<QuestInfo> newQuests) {
         quests = newQuests;
-        maybeUpdateTrackedQuest(trackedQuest);
         WynntilsMod.postEvent(new QuestBookReloadedEvent.QuestsReloaded());
     }
 
-    void updateMiniQuestsFromQuery(List<QuestInfo> newMiniQuests, QuestInfo trackedQuest) {
+    void updateMiniQuestsFromQuery(List<QuestInfo> newMiniQuests) {
         miniQuests = newMiniQuests;
-        maybeUpdateTrackedQuest(trackedQuest);
         WynntilsMod.postEvent(new QuestBookReloadedEvent.MiniQuestsReloaded());
     }
 
-    private void maybeUpdateTrackedQuest(QuestInfo trackedQuest) {
-        if (trackedQuest != this.trackedQuest) {
-            if (trackedQuest != null && trackedQuest.getName().equals(afterRescanName)) {
-                // We have stored the current task from last scoreboard update,
-                // now we can finally present it
-                trackedQuest.setNextTask(afterRescanTask);
-                afterRescanName = null;
-                afterRescanTask = null;
-                Models.Quest.updateTrackedQuest(trackedQuest);
-            }
-            WynntilsMod.warn("Tracked Quest according to scoreboard is " + this.trackedQuest + " but query says "
-                    + trackedQuest);
-        }
-    }
-
     void setDialogueHistory(List<List<StyledText>> newDialogueHistory) {
         dialogueHistory = newDialogueHistory;
         WynntilsMod.postEvent(new QuestBookReloadedEvent.DialogueHistoryReloaded());
     }
 
     /** Shared between the container query classes */
     public String getQuestBookTitleRegex(int pageNum) {
-        return "^§0\\[Pg. " + pageNum + "\\] §8.*§0 (Mini-)?Quests$";
+        return "§f\uE000\uE072";
```

Comment:
Weird!

Reply:
Yeah, odd. This was not really part of this fix; I started out trying to get the Quest Book working and it just slipped in. But it will be needed anyhow.